arsceneview/src/main/kotlin/io/github/sceneview/ar/interaction/ArNodeManipulator.kt

```diff
@@ -48,7 +49,9 @@ class ArNodeManipulator(sceneView: SceneView) : GestureHandler(sceneView), Manip
         currentGesture = gesture
         activeGesture = when (gesture) {
             GestureDetector.Gesture.ZOOM -> ScaleGesture(arNode)
-            GestureDetector.Gesture.ORBIT -> TranslationGesture(arNode)
+            GestureDetector.Gesture.ORBIT -> (arNode as? ArModelNode)?.let {
```

Comment:
@ThomasGorisse I require ArModelNode for translation now (and not only ArNode). Does it make sense to require ArModelNode for all gestures? What would be the case for ArNode?

Reply:
Sorry for the confusion.
In fact it should be an `ArNode`.

Reply:
Reverted